doc/message-registry.yaml

```diff
@@ -0,0 +1,136 @@
+# NOTE:
+# • A code should always have the same meaning forever.
+# • Codes may be retired or introduced in new releases.
+# See the following example of guidelines:
+# https://github.com/haskellfoundation/error-message-index/blob/main/tool-developers.md#code-assignment-recommendations
+
+# TODO: add precise guidelines
+# TODO: fix missing detailed description, examples, resolution
+
+- id: "unrecognized-keysym"
+  code: 107
+  added: UNKNOWN
+  type: warning
+  description: "Warn on unrecognized keysyms."
```

Comment:
feel free to ignore this but afaict for short sentences like that English doesn't usually use a full stop (but German for example does)

Reply:
Fixed